packages/maskbook/src/social-network-provider/facebook.com/UI/injectSetupPrompt.tsx

```diff
@@ -0,0 +1,37 @@
+import { LiveSelector, MutationObserverWatcher } from '@dimensiondev/holoflows-kit'
```

Comment:
On facebook, `<NotSetupYetPrompt />` and `< PostDialogHint />` need to be placed at different position, so injecting them respectively.